Makefile

```diff
@@ -5,6 +5,7 @@ BUILDTAGS=seccomp
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
 all:
+	ln -sfn $(CURDIR) $(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
 	go build -tags "$(BUILDTAGS)" -o runc .
```

Comment:
@tianon @gitido I'm not sure about the right way to add this in `Godeps` since #321 got some trouble, if we can add this symlink in repo, I'm happy to see merging that solution :)